source/_views/post.html

```diff
@@ -42,8 +42,15 @@
       <p class="tags">
       Tags:
       {% for tag in page.tags %}
-        <a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
-        {% if not loop.last %}, {% endif %}
+        {% if tag == "nikki-stevens" %} 
```

Comment:
hey so since this code is repeated in two places, it's going to be easier to make a global variable called "author tags" and then check if the tag is in_array and skip it if that's the case.